sdk/batch/Microsoft.Azure.Management.Batch/src/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Batch Management Library")]
 [assembly: AssemblyDescription("Provides management functions for Microsoft Azure Batch services.")]
 
-[assembly: AssemblyVersion("14.0.0.0")]
-[assembly: AssemblyFileVersion("14.0.0.0")]
+[assembly: AssemblyVersion("14.2.0.0")]
```

Comment:
According to the latest guide: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=signing-and-publishing-nuget-packges

> Nuget Package version change (in .csproj), AssemblyFileVersion change. If it's a major version update, AssemblyVersion will also need to be changed (in AssemblyInfo.cs)

So if it's just a minor upgrade, you don't necessarily need to change `AssemblyVersion`.

Reply:
I didn't make this change.

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/batch/Microsoft.Azure.Management.Batch/src/Properties/AssemblyInfo.cs#L12